app/static/css/style.css

```diff
@@ -1,64 +1,207 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&family=Rubik:ital,wght@0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
+
+:root {
+    /* P = Primary | S = Secondary | T = Tertiary */
+    --p-orange: #E84610;
+    --p-blue: #009FE3;
+    --p-grey: #4A4A4F;
+    --p-navy: #445261;
+    --p-red: #D63649;
+    --s-grey: #E6ECF0;
+    --s-yellow: #EFB920;
+    --s-pink: #FF5A60;
+    --s-teal: #23BBBB;
+    --t-grey: #E0E7FF;
+    --bg-grey: #F5F5F5;
+}
+
+/* ------------ Main Text Content ------------ */
+h1 {
+    font-family: 'Montserrat', sans-serif;
+}
+
+hr.divider {
+    width: 50px;
+    border-top: 3px solid var(--s-teal);
+    display: block;
+    margin: 0px auto;
+    background-color: var(--p-orange);
+    color: var(--p-orange);
+    height: auto;
+    opacity: 1;
+}
+
+/* ------------ Navigation ------------ */
+a,
 a.link-text {
     text-decoration: none;
-    color: rgb(41, 158, 158);
+    color: var(--s-teal);
+    font-weight: 500;
+    text-transform: uppercase;
 }
 
+a,
 a.link-text:hover {
-    color: rgb(16, 99, 99);
+    color: var(--p-navy);
 }
+/* ------------./end-navigation ------------ */
 
 
-/* Authentication Forms */
+/* ------------ Authentication Forms ------------ */
+h1.auth-header {
+    font-family: 'Montserrat', sans-serif;
+    font-weight: 600;
+    text-transform: uppercase;
+    font-size: 2rem;
+    text-shadow: 2px 2px 4px #00000047;
+}
 
 #registerForm {
-    align-items: center;
-    padding-top: 40px;
-    padding-bottom: 40px;
+    font-family: 'Montserrat', sans-serif;
+    font-weight: 400;
 }
 
-main.form-register,
-main.form-login {
-    display: block;
+main.container-fluid.p-3 {
+    /* background-color: var(--bg-grey); */
+    min-height: 100vh;
+    overflow: hidden;
+    margin: 0px auto;
```

Comment:
And here, remove the `px` after the zero.

Reply:
ah yes, thank you for pointing this out!